### DISCUSSION TITLE
Image resizing and padding

### DISCUSSION THREAD

**Discussion by khanonenet** [ultralytics/ultralytics#2303] [Q&A]
I am starting out at Yolov8 and I need help. I have trained a custom Yolov8 and had resized my training images to 640x640 using Roboflow. I wan to know if YOLOV8 resizes the images to the required input size on its own when training or do I have to manually resize them. And if I have to manually resize them can some one guide me how to do so? My image sizes are 1920x1080. Thank you

**Comment by aakash-chakraborty1995** [comment]
During training you can resize them automatically. 

# Train the model
model.train(data='coco128.yaml', epochs=100, imgsz=640)

**Reply by glenn-jocher** [reply]
@aakash-chakraborty1995 yOLOv8 automatically resizes and pads images during training to match the input size specified in your training command. There's no need for you to manually resize images to 640x640 if you're using YOLOv8, as the model will handle this for you. Just ensure that your dataset is correctly labeled and formatted, and YOLOv8 will take care of the rest during the training process. If your images are 1920x1080, YOLOv8 will resize them to the `imgsz` you specify in your training command. Happy training! 😊🚀
